src/components/ArticleGrid/index.tsx

```diff
@@ -24,26 +24,34 @@ export default class ArticleGrid extends React.Component<ArticleGridProps> {
       return <ArticleCard article={article} key={i} />
     })
 
+    const gridStyles = this.props.articles.length > 1 ? `
+      display: grid;
+      grid-template-columns: repeat(${this.props.maxColumns}, 1fr);
+      grid-gap: 16px;
+    ` : `
+      text-align: center;
+    `;
+
     return (
       <section
         id={!!this.props.heading ? slugify(this.props.heading) : undefined}
+        className={css`
+          margin-top: 20px;
+        `}
       >
         {!!this.props.heading && (
           <h2
             className={css`
               text-align: center;
+              font: bold 4rem Garamond, Georgia, sans-serif;
```

Comment:
garamond isn't web safe, let's go with [EB Garamond](https://fonts.google.com/specimen/EB+Garamond).

we'll wanna import via typography.js